deploy/ansible/terraform/gcp/terraform.yml

```diff
@@ -5,80 +5,26 @@
     force_init: true
     variables:
       project: "{{project}}"
-      uniqifier: "{{uniqifier}}"
       ssh_user: "{{ssh_user}}"
       ssh_key: "{{ssh_key}}"
+      node_count: "{{node_count|default(3)}}"
   register: terraform_out
 
-- name: Add sf-1 to ansible with a ssh key
+- name: Add sf nodes
   add_host:
-    hostname: sf-1
-    ansible_ssh_host: "{{terraform_out.outputs.sf_1_external.value[0]}}"
-    ansible_ssh_private_key_file: "{{ssh_key_filename}}"
+    hostname: "sf-{{node_idx+1}}"
+    node_name: "sf-{{node_idx+1}}"
+    ansible_ssh_host: "{{item}}"
+    node_ip: "{{item}}"
+    ansible_ssh_private_key_file: "{{ssh_key_filename|default(omit)}}"
     groups: hypervisors, primary, etcd_master
```

Comment:
So this is a subtle thing in gcp-xl which has been missed... Only the first three nodes over there are etcd_masters.